src/main/java/DateAndTime.java

```diff
@@ -0,0 +1,34 @@
+public class DateAndTime {
```

Comment:
Very intuitive way of processing date and time. I like it! Moving on, checks should be made to prevent invalid date/time inputs from being processed eg. 13/16/19 2460

Reply:
Thanks for the inputs!